scipy/stats/_continuous_distns.py

```diff
@@ -6123,6 +6125,43 @@ def _stats(self, b, moments='mv'):
     def _entropy(self, c):
         return 1 + 1.0/c - np.log(c)
 
+    def fit(self, data, *args, **kwds):
+        data = np.asarray(data)
+
+        floc = kwds.pop('floc', None)
+
+        # borrowed from distn_infrastructure
+        # puts shape into f0
+        if self.shapes:
+            shapes = self.shapes.replace(',', ' ').split()
+            for j, s in enumerate(shapes):
+                key = 'f' + str(j)
+                names = [key, 'f' + s, 'fix_' + s]
+                val = _get_fixed_fit_value(kwds, names)
+                if val is not None:
+                    kwds[key] = val
+
+        shape = kwds.pop('f0', None)
+        fscale = kwds.pop('fscale', None)
+
+        _check_fit_input_parameters(data, args,
+                                    kwds, fixed_param=(shape, floc, fscale))
+
+        if floc is None or fscale is not None or np.any(data - floc < 0):
+            if floc is not None:
+                kwds['floc'] = floc
+            if fscale is not None:
+                kwds['fscale'] = fscale
+            if shape is not None:
+                kwds['f0'] = shape
+            return super(pareto_gen, self).fit(data, *args, **kwds)
+        _check_unknown_kwds(kwds)
+        data = data - floc
+        fscale = np.min(data)
+        if shape is None:
```

Comment:
Did we decide that this is one of those distributions where the optimal `scale` is independent of the `shape`? (In other words, we decided that `fscale = np.min(data)` is the optimal `scale` regardless of what the use picks for `shape`?) If so, note why here.

Reply:
I observed this behavior, but I don't have a mathematical reason for it. I did incorporate it into the tests. 
 https://github.com/swallan/scipy/blob/339e93992ba5ff618319bf7be2508971dce28322/scipy/stats/tests/test_distributions.py#L987-L995

Reply:
@mdhaber, I've been looking `pareto` today, and I'm in the middle of editing a longish comment.  I'll have more soon.